doc/release-process.md

```diff
@@ -1,24 +1,11 @@
 # Release Process
 
-The rationale for this specific release process is:
+The detailed Release Process is explained [here](https://input-output.atlassian.net/wiki/spaces/REL/pages/1688174696/Release+Process+steps).
```

Comment:
Any specific reason to have private links on public repo?

Reply:
Yeah, that should not be there.